keycloak/keycloak-admin-ui
# User credentials

## Motivation
Adding "Credentials" tab for Users section as per the following: 
1. Empty state, set password, delete password -> https://marvelapp.com/prototype/a9b33db/screen/83342134
2. Reset password -> https://marvelapp.com/prototype/a9b33db/screen/83342111
3. View data -> https://marvelapp.com/prototype/a9b33db/screen/83342116

Note: (1) Multiple credentials and (2) change user label designs haven't been implemented yet. I need to get some clarifications before proceeding. Also tests will be added in a separate PR.

## Verification Steps
1. Navigate to Users screen and add a new user. Select created user and navigate to 'Credentials" tab.
2. Verify empty state and set a new password. Complete the form. Both passwords should match (if not error should show - not implemented yet). For now, only an alert is displaying and preventing the user from setting up a new password. Once set up successfully, the password should display in a table.
3. Select "Show data" and verify that password data dialog with data in alphabetical order is displaying. Close the dialog.
4. Select "Reset Password" and proceed with the reset. 
5. Select "Delete" from the dropdown and delete the password. Verify the empty state. 

## Checklist:
